risk_models/claus/claus.py

```diff
@@ -14,87 +14,107 @@
 VALID_MAX_AGE = 79
 
 
-def calculate_risk(
-        patient_age,
-        mother_onset_age=None,
-        daughter_onset_ages=None,
-        full_sister_onset_ages=None,
-        maternal_aunt_onset_ages=None,
-        paternal_aunt_onset_ages=None,
-        maternal_grandmother_onset_ages=None,
-        paternal_grandmother_onset_ages=None,
-        maternal_half_sister_onset_ages=None,
-        paternal_half_sister_onset_ages=None):
+def calculate_risk(patient_age,
+                   mother_onset_age=[],
+                   daughter_onset_ages=[],
+                   full_sister_onset_ages=[],
+                   maternal_aunt_onset_ages=[],
+                   paternal_aunt_onset_ages=[],
+                   maternal_grandmother_onset_ages=[],
+                   paternal_grandmother_onset_ages=[],
```

Comment:
Mutable list as default argument has interesting behavior in python https://docs.python-guide.org/writing/gotchas/